mscore/instrwidget.cpp

```diff
@@ -426,7 +426,7 @@ PartListItem::PartListItem(Part* p, QTreeWidget* lv)
       part = p;
       it   = 0;
       op   = ListItemOp::KEEP;
-      _name = QString(p->instrument()->trackName());
+      _name = QString(p->partName());
```

Comment:
Mayby use `instrument->trackname()` as the fallback it `partname()` is empty?

Reply:
It crossed my mind but I reverted to what it used to be in 3.5.
Nevertheless a good idea if you don't mind to deviate from what it is used to be :wink:

Reply:
The better is good's enemy ;-)

Reply:
It is by the way now exactly as it is in MU4 already. If we change it, then it should be changed in MU4 too. Which could be a nice challenge of course...

Reply:
Yep

Reply:
Better than potentially an empty sting there

Reply:
Done.

Reply:
thanks